src/react-apps/applications/ux-editor/src/containers/Toolbar.tsx

```diff
@@ -265,12 +263,12 @@ class ToolbarClass extends React.Component<IToolbarProps, IToolbarState> {
               disableUnderline: true,
               endAdornment:
                 <InputAdornment position={'end'} classes={{ root: classNames(this.props.classes.searchBoxIcon) }}>
-                  <i className={'ai ai-search'} />
+                  <i className={'fa fa-search'} />
                 </InputAdornment>,
               classes: { root: classNames(this.props.classes.searchBoxInput) },
             }}
           />
-        </FormControl>
+        </FormControl> */}
```

Comment:
Why is this commented out?

Reply:
Search in toolbar is currently not supported, but it will be at a later time.